Snow-Media-Technologies/filmo-backend
# FILMO-014-story(SOCIAL-MEDIA-LOGIN): Enable social media login

## Description

- Configure social media login
- Implement login with googleplus
- Implement login with twitter
- Implement login with facebook
- Implement login with linkedin

## Type of change
Please select the relevant option


## How Has This Been Tested?
Please describe the tests that you ran to verify your change

## How to Manually Test
- Checkout to this branch
- Configure your ```.env``` file as shown in the ```.envSample```
- Run the tests to make sure your are all set with ```npm test```
- Run ```npm start```
- Go to the user routes folder and start interacting with available routes

## Checklist:
